addons/account_check_printing/tests/test_print_check.py

```diff
@@ -177,3 +177,14 @@ def test_print_great_pre_number_check(self):
         payment.action_post()
         self.assertTrue(payment.write({'check_number': '2147483647'}))
         self.assertTrue(payment.write({'check_number': '2147483648'}))
+
+        payment = self.env['account.payment'].create({
+            'payment_type': 'outbound',
+            'partner_type': 'supplier',
+            'amount': 100.0,
+            'journal_id': self.company_data['default_journal_bank'].id,
+            'payment_method_id': self.payment_method_check.id,
+        })
```

Comment:
```suggestion
        vals = {
            'payment_type': 'outbound',
            'partner_type': 'supplier',
            'amount': 100.0,
            'journal_id': self.company_data['default_journal_bank'].id,
            'payment_method_id': self.payment_method_check.id,
        }
        payment = self.env['account.payment'].create(vals)
```
Maybe put the the payment values in a variable, so you can reuse them :wink:

Reply:
Done :)